db/schema.rb

```diff
@@ -178,9 +178,6 @@
     t.index ["uuid"], name: "index_courses_on_uuid", unique: true
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
We've decided we want to keep this in the schema. Can you un-delete-if-icate it?